### PR TITLE
Fix: login-logo URL can be incorrect for all the sub-sites in a network

### LINKED ISSUE #259: Login logo is 404 for sub-sites
Steps to reproduce:
1. Open a login page for any sub-sites
2. Login Logo will not be visible and the logo URL will be 404 in the developer console.

This is probably because the login logo URL that is generated is dynamically generated based on the site URL of the sub-sites 

https://github.com/humanmade/altis-cms/blob/4271a71d0e715a4977950e715f264393203e114b/inc/namespace.php#L185

Acceptance criteria:

- [ ]  Login logo is visible on all sub-sites

### PR DESCRIPTION
**Pull request by Nikschavan**
Fix #259

### CODE REVIEW

**Comment by Nikschavan** [comment]
Wondering if this will be backported to v4 as well?

File: inc/namespace.php [hunk 1]
```diff
@@ -182,7 +182,7 @@ function add_login_logo() {
 	?>
 	<style>
 		.login h1 a {
-			background-image: url('<?php echo esc_url( site_url( $logo ) ); ?>');
+			background-image: url('<?php echo esc_url( get_site_url( 1, $logo ) ); ?>');
```
Comment (roborourke) [inline inc/namespace.php:185]: Can this use `get_main_site_id( get_main_network_id() )` instead? I think 1 is generally correct but in some really bizarre circumstances it's not always the case

Comment (roborourke) [reply inc/namespace.php:185]:
```suggestion
			background-image: url('<?php echo esc_url( get_site_url( get_main_site_id( get_main_network_id() ), $logo ) ); ?>');
```

**Review by roborourke — CHANGES_REQUESTED** [review]
Minor update, thanks for doing this!

**Review by roborourke — APPROVED** [review]

**Pull request merged**